crates/provider/src/provider.rs

```diff
@@ -995,10 +996,35 @@ impl<N: Network, T: Transport + Clone> Provider<N, T> for RootProvider<N, T> {
     }
 }
 
+/// Admin namespace rpc interface that gives access to several non-standard RPC methods.
+#[cfg_attr(target_arch = "wasm32", async_trait::async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait::async_trait)]
```

Comment:
async_trait is no longer required, but we need to do use this macro https://github.com/alloy-rs/alloy/blob/f7333c4344fdf6a16f620bc4392ba81b64684017/crates/pubsub/src/connect.rs#L18-L18

Reply:
Done. Should I remove the usage of `async_trait` from `provider` and `rootProvider` too?

Reply:
not yet